templates/guide/verify_accuracy.html

```diff
@@ -0,0 +1,55 @@
+{% extends "_base.html" %}
+{% block page_title %}{{ feature.name }} - {% endblock %}
+
+{% block css %}
+<link rel="stylesheet" href="/static/css/forms.css?v={{app_version}}">
+<link rel="stylesheet" href="/static/css/guide.css?v={{app_version}}">
+{% endblock %}
+
+{% block subheader %}
+<div id="subheader">
+  <h2 id="breadcrumbs">
+    <a href="/guide/edit/{{ feature_id }}">
+      <iron-icon icon="chromestatus:arrow-back"></iron-icon>
+      Verify feature data for {{ feature.name }}
+    </a>
+  </h2>
+</div>
+{% endblock %}
+
+{% block content %}
+<form name="feature_form" method="POST" action="{{current_path}}">
+  <input type="hidden" name="token" value="{{xsrf_token}}">
+  {% for section_name, form in forms %}
+    <h3>{{ section_name }}</h3>
+    <section class="flat_form">
+      <chromedash-form-table>
+        {{ form }}
+      </chromedash-form-table>
+    </section>
+  {% endfor %}
+
+  <section class="final_buttons">
+    <chromedash-form-table>
+      <chromedash-form-field>
+        <span slot="field">
+            <input class="button" type="submit" value="Submit">
```

Comment:
@jrobbins, this is currently implemented to navigate back to the feature edit page after submission like the other edit pages. Will this suffice? Or would it be better to display a message confirming the user has updated the data?

Reply:
Navigating back to the edit page is fine.

Ideally we would display a confirmation as a toast or banner after any form submission.  I'll track an issue for that so that we implement it after these pages are converted to be SPA pages.  https://github.com/GoogleChrome/chromium-dashboard/issues/2092.